src/main/java/org/kiwiproject/dropwizard/util/server/DropwizardConnectors.java

```diff
@@ -106,10 +108,8 @@ private static HttpConnectorFactory getConnectorFactory(ConnectorType connectorT
     static Map<ConnectorType, ConnectorFactory> createConnectorFactoryMap(List<ConnectorFactory> connectors) {
         return connectors.stream()
                 .filter(factory -> factory instanceof HttpConnectorFactory)
-                .collect(toMap(
-                        factory -> ConnectorType.forClass(factory.getClass()),
-                        identity()
-                ));
+                .map(HttpConnectorFactory.class::cast)
+                .collect(toMap(ConnectorType::forHttpConnectorFactory, identity()));
```

Comment:
So this will blow up if the `connectors` list has more than one connector for the same `connectorType`

Reply:
Didn't test...will do that...

Reply:
Actually, what should we do in that case?

Reply:
We could use a merge function and just select one of them...

Reply:
I don't know.  Before this change we selected the first one.  So we either need to 1. limit to one or 2. change to a multimap.

1. version stays at 0.1.2
2. version bumps to 0.2.0 and we have to figure out how to handle multiples